threedi_api_qgis_client/widgets/wizard.py

```diff
@@ -23,6 +23,49 @@
 CONSTANT_RAIN = "Constant"
 CUSTOM_RAIN = "Custom"
 DESIGN_RAIN = "Design"
+AREA_WIDE_RAIN = {
+    '0': [0.0],
+    '1': [0.0],
+    '2': [0.0],
+    '3': [0.30, 0.60, 0.90, 1.50, 2.10, 2.10, 1.50, 1.20,
+          1.05, 0.90, 0.75, 0.60, 0.45, 0.30, 0.15],
+    '4': [0.15, 0.30, 0.45, 0.60, 0.75, 0.90, 1.05, 1.20,
+          1.50, 2.10, 2.10, 1.50, 0.90, 0.60, 0.30],
+    '5': [0.30, 0.60, 1.50, 2.70, 2.70, 2.10, 1.50, 1.20,
+          1.05, 0.90, 0.75, 0.60, 0.45, 0.30, 0.15],
+    '6': [0.15, 0.30, 0.45, 0.60, 0.75, 0.90, 1.05, 1.20,
+          1.50, 2.10, 2.70, 2.70, 1.50, 0.60, 0.30],
+    '7': [0.6, 1.2, 2.1, 3.3, 3.3, 2.7, 2.1, 1.5, 1.2, 0.9, 0.6, 0.3],
+    '8': [0.3, 0.6, 0.9, 1.2, 1.5, 2.1, 2.7, 3.3, 3.3, 2.1, 1.2, 0.6],
+    '9': [1.5, 2.7, 4.8, 4.8, 4.2, 3.3, 2.7, 2.1, 1.5, 0.9, 0.6, 0.3],
+    '10': [1.8, 3.6, 6.3, 6.3, 5.7, 4.8, 3.6, 2.4, 1.2],
+    '11': [5.833333333] * 12,   # DPRA lokaal T100
+    '12': [7.5] * 12,           # DPRA lokaal T250
+    '13': [6.666666667] * 24,   # DPRA lokaal T1000
+    '14': [0.208333333] * 576,  # DPRA regionaal T100
+    '15': [0.225694444] * 576,  # DPRA regionaal T250
+    '16': [0.277777778] * 576   # DPRA regionaal T1000
```

Comment:
i am pretty sure this is not english comment :)

Reply:
Indeed - comments removed.